src/main/java/org/jabref/logic/importer/fetcher/JstorFetcher.java

```diff
@@ -133,4 +169,9 @@ public String getName() {
     public TrustLevel getTrustLevel() {
         return TrustLevel.META_SEARCH;
     }
+
+    @Override
+    public void doPostCleanup(BibEntry entry) {
+        // do nothing
+    }
```

Comment:
Why did you overwrite this?

Reply:
as both SearchBased and IdBased Fetchers have a doPostCleanup default impl, the compiler is not sure which one to call